test/scan/scanPerf.graph

```diff
@@ -0,0 +1,4 @@
+perfkeys: took
+files: scanPerf.dat
```

Comment:
You may want to add a `graphkeys: <key>`. By default the key legend will be "took", which probably isn't that helpful.  Maybe `graphkeys: 1D scan` or something (`graphkeys: single scan 1d` would match the reduction graph)